app/templates/main/request-received.html

```diff
@@ -0,0 +1,37 @@
+{% extends "base.html" %}
+{% block beforeContent %}
+{{ super() }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-panel govuk-panel--confirmation">
+            <h1 class="govuk-panel__title">
+                Request received
+            </h1>
+        </div>
+        <h2 class="govuk-heading-m govuk-!-margin-top-7">
+            What happens next
+        </h2>
+        <p class="govuk-body">
+            We will email a link to <span class="govuk-!-font-weight-bold">{{ user_email }}</span>.
+        </p>
+        <p class="govuk-body">
+            This may take anything up to 5 minutes to come through to your inbox.
+        </p>
+        <h2 class="govuk-heading-m">
+            If you’ve not recieved the email
+        </h2>
+        <p class="govuk-body">
+            If you cannot see the email in your inbox, check your junk or spam folder.
+            If you’re still having problems,
+            <a class="govuk-link govuk-link--no-visited-state" href="url_for('download')">
+                request a new link to download your data
+            </a> or email us at
```

Comment:
```suggestion
            <a class="govuk-link govuk-link--no-visited-state" href="url_for('download')">request a new link to download your data</a>
            or email us at
```

The underline link styling is carrying on for an extra space because of the new line between the tags here 😬  be nice to clean that up as well

### Before
<img width="98" alt="image" src="https://github.com/communitiesuk/funding-service-design-post-award-data-frontend/assets/2920760/c30f70df-b289-42e7-b713-1dfe0eb1d076">


### After
<img width="91" alt="image" src="https://github.com/communitiesuk/funding-service-design-post-award-data-frontend/assets/2920760/1208a9c5-6c2f-4e51-97ec-5f0bb8cb730b">